src/catimg.c

```diff
@@ -67,14 +71,39 @@ int main(int argc, char *argv[])
     int c;
     opterr = 0;
 
-    uint32_t cols = 0, precision = 0;
+    uint32_t cols = 0, rows = 0, precision = 0;
+    uint32_t maxCols = 0, maxRows = 0;
```

Comment:
let's use snake case since everything is in snake case

Reply:
*after looking up what that is* sure I'll correct that with the other suggested changes